HW02/Mikhail_Sauchuk/mikhail_sauchuk_hw_02_t_04.rb

```diff
@@ -0,0 +1,19 @@
+# An arbitrary string is given, the method `task_4`
+# must return the count of letters and digits in that string.
+
+def count_letters(str)
```

Comment:
Sounds more clear for me
```suggestion
def letters_count(str)
```

Reply:
Got it, and have renamed